src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -1203,9 +1198,28 @@ public bool StatusCanHaveBody(int statusCode)
                    statusCode != 304;
         }
 
-        public void RejectRequest(string message)
+        private void ThrowResponseAlreadyStartedException(string value)
+        {
+            throw new InvalidOperationException(value + " cannot be set, response had already started.");
+        }
+
+        private void ThrowResponseAbortedException()
+        {
+            throw new ObjectDisposedException(
+                    "The response has been aborted due to an unhandled application exception.",
+                    _applicationException);
```

Comment:
gigantic nit: there are 8 spaces in these 2 lines - please indent with 4 spaces.